pwa/src/styling/design-tokens/component-overrides.css

```diff
@@ -277,6 +277,8 @@
   --denhaag-alert-content-margin-inline-end: var(--web-app-size-lg);
   --utrecht-heading-4-distanced-margin-block-end: var(--web-app-size-2xs);
   --denhaag-alert-action-padding-block-start: var(--web-app-size-md);
+  --denhaag-breadcrumb-color: var(--web-app-breadcrumb-color);
+  --denhaag-breadcrumb-current-color: #65757b;
```

Comment:
Is `#65757b` de kleur voor Rotterdam? Dan moet die op `.rotterdam-theme` geplaatst worden...

Reply:
Dit zou `var(--rotterdam-color-gray-tint-09)` moeten zijn. Zoek altijd even hier de kleuren op, om te kijken of er een token is in het design system: https://nl-design-system.github.io/themes/?path=/story/rotterdam-color--page